src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1889,19 +1887,19 @@ public TransformReason get(EntityTransformEvent event) {
 		// BellRingEvent - these are BlockEvents and not EntityEvents, so they have declared methods for getEntity()
 		if (Skript.classExists("org.bukkit.event.block.BellRingEvent")) {
 			EventValues.registerEventValue(BellRingEvent.class, Entity.class, new Getter<Entity, BellRingEvent>() {
-                @Override
-                @Nullable
-                public Entity get(BellRingEvent event) {
-                    return event.getEntity();
-                }
-            }, EventValues.TIME_NOW);
+				@Override
+				@Nullable
+				public Entity get(BellRingEvent event) {
```

Comment:
```suggestion
				public @Nullable Entity get(BellRingEvent event) {
```

Reply:
I understand why you want me to change it, but why am I changing something that wasn't added via this PR?
It only says I've edited it because of when I made sure to convert all spaces to tabs.

Reply:
It's still good to fix things according to conventions when you go past.